examples/porter-discourse/README.md

```diff
@@ -0,0 +1,62 @@
+# Discourse Example Bundle
+
+## Why Discourse?
```

Comment:
Let's put right at the top before we even say Why Discourse that this is a work in progress and is not a finished example bundle.